Formula/jdupes.rb

```diff
@@ -0,0 +1,18 @@
+class Jdupes < Formula
+  desc "Duplicate file finder and an enhanced fork of 'fdupes'"
+  homepage "https://github.com/jbruchon/jdupes"
+  url "https://github.com/jbruchon/jdupes/archive/v1.9.tar.gz"
+  sha256 "7f2505571c9fc8f76609b918106785f61e4a85af184309f738d7d847f1c2a1cf"
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    touch "a"
+    touch "b"
+
+    dupes = shell_output("#{bin}/jdupes -z .").strip.split("\n").sort
```

Comment:
neither the `.strip` nor the `.sort` is needed

Reply:
You are right, I am not really satisfied with my test as well. because of my total unfamiliarity with ruby I had just quickly copied the test from fdupes (https://github.com/Homebrew/homebrew-core/blob/master/Formula/fdupes.rb).

I will look at making a better test, and then probably suggest it for use in fdupes as well, this will take a while time though, I'd have to read up on ruby.

Reply:
@jd-a @ilovezfs I maintain a tap with jdupes in it. Here's my formula, including the requested test for uniqueness. Please free free to copy it or crib from it as needed if it helps.

https://github.com/mckern/homebrew-quality-of-life/blob/master/jdupes.rb

Reply:
Thanks @mckern!